src/models/developmentAccount.ts

```diff
@@ -0,0 +1,124 @@
+    return new Application(application, this.sessionHandler)
+  }
+
+  async getApplications (options: { sort?: ApplicationSorting, ascending?: boolean, limit?: number, offset?: number }) {
+    const { applications, applicationsMeta } = (await this.action('application:list', { ...options, developmentAccountId: this.id })) as ApplicationListResponse
+    return new Applications(applications, applicationsMeta, this.sessionHandler)
+  }
+
+  async initDevelopmentAccountRelationshipRequest (params: { email: string, role: DevelopmentAccountRelationshipRole }) {
+    const { developmentAccountRelationshipRequest } = (await this.action('developmentAccountRelationshipRequest:init', { ...params, developmentAccountId: this.id })) as DevelopmentAccountRelationshipRequestResponse
+    return new DevelopmentAccountRelationshipRequest(developmentAccountRelationshipRequest, this.sessionHandler)
+  }
+
+  async getDevelopmentAccountRelationshipRequest (params: { id: number }) {
+    const { developmentAccountRelationshipRequest } = await this.action('developmentAccountRelationshipRequest:show', { ...params, developmentAccountId: this.id }) as DevelopmentAccountRelationshipRequestResponse
+    return new DevelopmentAccountRelationshipRequest(developmentAccountRelationshipRequest, this.sessionHandler)
+  }
+
+  async getDevelopmentAccountRelationshipRequests (options: { email?: string, sort?: DevelopmentAccountRelationshipRequestSorting, ascending?: boolean, limit?: number, offset?: number}) {
+    const { developmentAccountRelationshipRequests, developmentAccountRelationshipRequestsMeta } = await this.action('developmentAccountRelationshipRequest:list', { ...options, developmentAccountId: this.id }) as DevelopmentAccountRelationshipRequestListResponse
+    return new DevelopmentAccountRelationshipRequests(developmentAccountRelationshipRequests, developmentAccountRelationshipRequestsMeta, this.sessionHandler)
+  }
+
+  async getDevelopmentAccountRelationship (params: { id: number }) {
+    const { developmentAccountRelationship } = await this.action('developmentAccountRelationship:show', { ...params, developmentAccountId: this.id }) as DevelopmentAccountRelationshipResponse
+    return new DevelopmentAccountRelationship(developmentAccountRelationship, this.sessionHandler)
+  }
+
+  async getDevelopmentAccountRelationships (options?: { sort?: DevelopmentAccountRelationshipSorting, ascending?: boolean, limit?: number, offset?: number}) {
+    const { developmentAccountRelationships, developmentAccountRelationshipsMeta } = await this.action('developmentAccountRelationship:list', { ...options, developmentAccountId: this.id }) as DevelopmentAccountRelationshipListResponse
+    return new DevelopmentAccountRelationships(developmentAccountRelationships, developmentAccountRelationshipsMeta, this.sessionHandler)
+  }
+
+  async update (options: { company?: string, address?: string, websiteUrl?: string }) {
```

Comment:
You list these parameters as `optional` here, but they are not `nullable` on the model, so they should be required here too.